packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -2265,13 +2371,13 @@ curl -v -X PUT \
 HTTP/1.1 204 No Content
 ```
 
-##### Add Group target to group administrator role given to a group
+##### Replace a Group target in a group administrator role given to a group
```

Comment:
@vasishtajayanti-okta Should this be "Assign" or "Create", in this case?

Reply:
This should also be "Assign"